index.html

```diff
@@ -156,128 +156,14 @@ <h2 class="jobtitle">Software Developer</h2>
               
```

Comment:
- [ ] You did great work on this project but you still need to work on something for your project to look more professional, when running your project we can't see the popup for the work section for both mobile and desktop version, check the screenshot below, kindly fix this :point_down: 
![popupnot](https://user-images.githubusercontent.com/88894525/204051432-f1e2438a-1bbd-453c-bed0-c02b7b33b9ef.png)

Reply:
![image](https://user-images.githubusercontent.com/91262542/204053941-558f2d29-62c9-463c-8435-92233bdd573b.png)

![image](https://user-images.githubusercontent.com/91262542/204054158-0df29feb-d141-41ee-932e-da4d10857af4.png)

At my end  the project loads well